.travis.yml

```diff
@@ -21,6 +21,11 @@ script:
   - docker exec "$(cat container_id)" make check
   - docker exec "$(cat container_id)" make coverage
 
+after_success:
+  # Analyze the code coverage.
+  - docker exec "$(cat container_id)" coverage3 xml -i
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
This is super weird to run code from the web. However, if they have it as recommended way...

Reply:
Yes, it is a preferred way.